src/main/java/org/opentestsystem/ap/imrt/common/repository/ImrtItemRepository.java

```diff
@@ -0,0 +1,15 @@
+package org.opentestsystem.ap.imrt.common.repository;
```

Comment:
Copied from ingest service.  I considered copying more but at this time the ISS only needs to get item by id.